### DISCUSSION TITLE
tutorials/performance/general_optimization

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#277] [User-contributed notes]
# tutorials/performance/general_optimization

Introduction: In an ideal world, computers would run at infinite speed. The only limit to what we could achieve would be our imagination. However, in the real world, it's all too easy to produce so...

https://docs.godotengine.org/en/stable/tutorials/performance/general_optimization.html

**Comment by Ataraxia-Mechanica** [comment]
https://sampruden.github.io/posts/godot-is-not-the-new-unity/
I think this article should mention how slow raycast in code is. Basically you should only use raycast nodes if you want to do more than a few checks per frame which kinda suck
